docs/37-actions.mdx

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This should probably be after components (components are the star), and even though they really are called `Actions`, we should probably name them something less generic here to reduce confusion with components. Maybe something like `Animation Actions` or `Trigger Actions` or something.

Reply:
I feel like `Actions` is not that confusing. There is also a brief description on that page about what actions are and every component using them calls them `Actions`.